### ISSUE TITLE
Add `setDrawing` and `setDrawingSvg` methods

### ISSUE THREAD

**Issue by kenny-evitt** [bradmartin/nativescript-drawingpad#50]
The *title* of issue #32 doesn't match my request by the description does:

> If I need to "edit" a signature can I get the old signature in order to see the old one?

I don't want the "old signature" as a 'background'. I just want to load a previous drawing (from an SVG).

**Comment by kenny-evitt** [comment]
I'm particularly interested in this for iOS. I added an issue to the upstream project:

 - [Add methods to set or load an existing signature · Issue #11 · michalkonturek/SignatureView](https://github.com/michalkonturek/SignatureView/issues/11)

**Comment by kenny-evitt** [comment]
Unfortunately, the upstream iOS project has been inactive for about 5 years.

**Comment by govi2010** [comment]
@kenny-evitt  did you figure out ?

**Comment by kenny-evitt** [comment]
@govi2010 Nope. I never received a reply or comment on the upstream issue and I haven't looked into what changes would need to be made to the upstream project either.

**Comment by govi2010** [comment]
I have asked my IOS team to look into it and get it done for IOS as soon as
possible.

Do you have Android resources which can update upstream project?

Once this is done, I can update Nativescript project.

**Comment by kenny-evitt** [comment]
@govi2010 Great news about the iOS support.

This issue seems to imply that this can already be done in the upstream Android project:

 - [Loading existing signature from SVG text · Issue #100 · gcacace/android-signaturepad](https://github.com/gcacace/android-signaturepad/issues/100)

**Comment by govi2010** [comment]
Great. Team is working on IOS.

**Comment by san3k1990** [comment]
Is it possible to edit the image using this plugin? Or is it going to be implemented in the future?
